packages/headless/src/api/generated-answer/generated-answer-event-payload.ts

```diff
@@ -0,0 +1,33 @@
+export enum GeneratedAnswerStreamFinishReason {
```

Comment:
A bit nitpicky, but I generally find string enum un-necessary in TypeScript when you can simply use string literals.

ie:
```typescript
export type GeneratedAnswerStreamFinishReason = 'COMPLETED' | 'ERROR';
```

Similar comment for `GeneratedAnswerPayloadType`

Reply:
>  i'm just usually weary of typing constant strings out like this.

vs code auto complete will work no problem 😂